third-party/realdds/include/realdds/topics/device-info-msg.h

```diff
@@ -12,14 +11,23 @@ namespace topics {
 
 class device_info
 {
+    nlohmann::json _json;
+
 public:
-    std::string name;
-    std::string serial;
-    std::string product_line;
-    std::string topic_root;
-    bool locked = true;
+    //std::string serial;
+    //std::string product_line;
+    //bool locked = true;
+
+    std::string const & name() const;
+    void set_name( std::string && );
```

Comment:
I am not sure from user perspective that rvalue is always desired, for instance `rs2_device_to_info` copies the name

Reply:
Yeah I debated but eventually I just decided that it's fine if we copied before calling the function. Otherwise I can change to const &. What would you prefer?

Reply:
I think I prefer `const &`

Reply:
This means always a copy vs. a move if possible. But OK, done.